recipes/kalpy/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "kalpy" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: kalpy
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/k/kalpy-kaldi/kalpy-kaldi-{{ version }}.tar.gz
+  sha256: 61266371b1e2efd69558f4f32585202c016753fa5fc0a17ada9df3958f1b9d24
+
+build:
+  script:
+    - CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY" {{ PYTHON }} -m pip install . --no-deps -vv   # [not win]
+    - {{ PYTHON }} -m pip install . --no-deps -vv                                                         # [win]
+  number: 0
+  skip: True  # [py<38]
+  script_env:
+   - KALDI_ROOT={{ environ.get('LIBRARY_PREFIX') }}  # [win]
+   - KALDI_ROOT=$PREFIX           # [not win]
```

`script_env` is not really so much in use. rather than work around bugs there (e.g. if `%LIBRARY_PREFIX%` doesn't work out of the box), it would IMO be easier to just add it to the script section.

```suggestion
  script:
   - export KALDI_ROOT="$PREFIX"        # [unix]
   - set "KALDI_ROOT=%LIBRARY_PREFIX%"  # [win]
    - CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY" {{ PYTHON }} -m pip install . --no-deps -vv   # [not win]
    - {{ PYTHON }} -m pip install . --no-deps -vv                                                         # [win]
  number: 0
  skip: True  # [py<38]
```